FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/AuthPickerView.swift

```diff
@@ -15,46 +15,50 @@ public struct AuthPickerView {
 
 extension AuthPickerView: View {
   public var body: some View {
-    VStack {
-      Text(authService.string.authPickerTitle)
-        .font(.largeTitle)
-        .fontWeight(.bold)
-        .padding()
-      if authService.authenticationState == .authenticated {
-        SignedInView()
-      } else if authService.authView == .passwordRecovery {
-        PasswordRecoveryView()
-      } else if authService.authView == .emailLink {
-        EmailLinkView()
-      } else {
-        if authService.emailSignInEnabled {
-          Text(authService.authenticationFlow == .login ? authService.string
-            .emailLoginFlowLabel : authService.string.emailSignUpFlowLabel)
-          Divider()
-          EmailAuthView()
+    ScrollView {
+      VStack {
+        if authService.authView == .authPicker {
```

Comment:
I made the "Welcome" title only appear when on `authPicker` view, other wise it appeared in the others, e.g. PasswordRecoveryView.

Reply:
If you intend on having a title for the forgot password view later on, you may want to abstract this into an `@ViewBuilder header` property or something.

Reply:
Made it a computed property: [`b4ea600` (#1259)](https://github.com/firebase/FirebaseUI-iOS/pull/1259/commits/b4ea600140900c6be8fb9609a398095b9b9f09ca)